src/documentation/programming-guide.md

```diff
@@ -1398,7 +1398,7 @@ If your `PCollection` is bounded (the size is fixed), you can assign all the ele
 %}
 ```
 
-### Time skew, data lag, and late data
+### <a name="watermarks-late-data"></a>Watermarks and late data
```

Comment:
Just curious - could we put the text in the `<a>` tag so that it is easy for users to link to a section? I feel like we can use CSS to make the visual and interactive behavior otherwise identical?